Disable INFORMATION_SCHEMA.SOCKET_DIAG_SLAVES table on macOS

laurynas-biveinis opened the pull request:
Because macOS does not have the required functionality.

Squash with accb01cc760345cba3f01776c997780935b6546b

yoshinorim commented:
This makes the socket_diag_slaves no longer tested from dd_is_compatibility_ci, is_tables_is etc. How about adding 	
"--source include/not_mac_os.inc" on these files and forking another test for mac?

laurynas-biveinis commented:
@yoshinorim that makes sense. Since macOS is not going to be a first class platform, and any changes go through Linux testing eventually anyway, I am thinking of just adding `--source include/not_mac_os.inc`, and leaving the forking of the test to indeterminate future

laurynas-biveinis committed:
Disable INFORMATION_SCHEMA.SOCKET_DIAG_SLAVES table on macOS

Because macOS does not have the required functionality.

Squash with accb01cc760345cba3f01776c997780935b6546b

laurynas-biveinis commented:
Ready for respin
